lib/werewolf/game.rb

```diff
@@ -322,12 +322,12 @@ def status()
     def format_time
       if active?
         if time_period == 'night'
-          "It is night (day #{day_number}).  The sun will rise again in #{time_remaining_in_round} seconds."
+          ":night_with_stars: It is night (day #{day_number}).  The sun will rise again in #{time_remaining_in_round} seconds. :hourglass:"
```

Comment:
i'm trying to keep all the slack-specific formatting out of `Game`.  i've been moving to a model where the observer gets a notify with all the info it needs, but with none of the messaging baked in.  this is fine to merge, but that's my thinking